docs/community/usingtokens.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Using your UMA tokens
+sidebar_label: How to use your UMA tokens?
+---
+
```

Comment:
Need a bit of an intro, how about

"So you've got some UMA....but what do you do with it?

Read on to find some suggestions for how you can use your UMA to participate in keeping our protocol secure, use it as collateral to mint dollar pegged expiring tokens or call options which entitle you to purchase more UMA at a fixed point int he future,   If you have held UMA for a while, its [worth checking](https://claim.umaproject.org/) if you received our uTVL airdrop which you can claim until the end of June.